### PR TITLE
docs: fix example in README.md. Encapsulate in async + other fixes

### PR DESCRIPTION
**Pull request by smoya**
**Description**

This PR fixes the example in `README.md` by:

- Encapsulate call to the generator in an async func.
- Add `console.log` for all the generated models.
- Show the real output of the example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,19 +93,35 @@ const doc = {
   required: ["street_name", "city", "state", "house_number", "state"],
 };
 
-const interfaceModel = await generator.generate(doc);
+async function generate() {
+  const models = await generator.generate(doc);
+  models.forEach(function (model) {
+    console.log(model.result);
+  });
```
Comment (magicmatatjahu) [inline README.md:100]:
```suggestion
  models.forEach(model => console.log(model.result)); 
```

File: README.md [hunk 2]
```diff
@@ -93,19 +93,35 @@ const doc = {
...
+  }); 
+}
+
+generate();
+```
 
-// interfaceModel[0] should have the shape:
-interface Address {
+Output:
+```js
```
Comment (magicmatatjahu) [inline README.md:107]:
```suggestion
Output:

```ts
```

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]
Overall good, but some needed changes :)

**Comment by smoya** [comment]
@all-contributors please add @smoya for doc and review

**Review by magicmatatjahu — APPROVED** [review]
LGTM! @jonaslagoni Do you want to make another review?

**Comment by jonaslagoni** [comment]
Don't need to no, go ahead! 🎉

**Pull request merged**